<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OdbcQueryDlg</class>
 <widget class="QDialog" name="OdbcQueryDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ODBC Query test</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../qHostMonitor.qrc">
    <normaloff>:/img/hostMonitor.png</normaloff>:/img/hostMonitor.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="gpDataSource">
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Data Source</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="lblDataSource">
          <property name="text">
           <string>Data source</string>
          </property>
          <property name="buddy">
           <cstring>cmbDataSource</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cmbDataSource">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblLogin">
          <property name="text">
           <string>login</string>
          </property>
          <property name="buddy">
           <cstring>cmbLogin</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cmbLogin">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblPassword">
          <property name="text">
           <string>Password</string>
          </property>
          <property name="buddy">
           <cstring>editPassword</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="editPassword"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lblTimeout">
          <property name="text">
           <string>Timeout</string>
          </property>
          <property name="buddy">
           <cstring>spinTimeout</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QSpinBox" name="spinTimeout">
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>300</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>sec</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QDialogButtonBox" name="btnOdbcBlock">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOdbcTest">
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Test</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="gpTest">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Test</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="chkTestAlert">
          <property name="text">
           <string>Alert when field in</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinAlertRow">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>row</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinAlertCol">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>col</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lblFieldUnavailable">
          <property name="text">
           <string>If field is unavailable, set status</string>
          </property>
          <property name="buddy">
           <cstring>cmbUnavailableStatus</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbUnavailableStatus">
          <item>
           <property name="text">
            <string>Unknown</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bad</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ok</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QComboBox" name="cmbCriteriaMode">
          <item>
           <property name="text">
            <string>is &lt; than</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>is &gt; than</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>is = to</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>is &lt;&gt; from</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>contains</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>does not contain</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editCriteriaValue"/>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="rbConnectionOnly">
        <property name="text">
         <string>Check connection only</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPlainTextEdit" name="txtSqlQuery"/>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="rbExecuteQuery">
        <property name="text">
         <string>Execute SQL query</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qHostMonitor.qrc"/>
  <include location="../../qHostMonitor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnOdbcBlock</sender>
   <signal>accepted()</signal>
   <receiver>OdbcQueryDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOdbcBlock</sender>
   <signal>rejected()</signal>
   <receiver>OdbcQueryDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
